docs/ds9.rst

```diff
@@ -39,9 +39,7 @@ Serialisation is done using the `~regions.ds9_objects_to_string` function
 
 .. warning::
 
-    The API for serialisation to DS9 string is currently not using the
-    `~regions.Shape` layer. It should be adapted.  Also, all regions currently
-    have ``meta`` and ``visual`` arguments for ``__init__`` and stored as
+    Also, all regions currently have ``meta`` and ``visual`` arguments for ``__init__`` and stored as
```

Comment:
Be sure to keep paragraphs wrapped at 80 characters, otherwise diffs get really ugly.